docs/UserGuide.md

```diff
@@ -71,7 +112,43 @@ open the help window.<br>
 
 ## Navigating the User Interface
 
-placeholder
+![NavigatingUi](images/NavigatingUserInterface.png)
+The user interface is split into the input and output sections, as well as
+two view columns for **Person List** and **Overview**.
+
+Item | Description
+--------|------------------
+**command box** | This is where you enter your commands to DengueHotspotTracker.
+**output box** | The output box displays additional information from a command. <br> :bulb: For a *successful* command, the output box may contain information such as the number of entries listed. <br> :bulb: For an *unsuccessful* command, such as invalid input, the output box will give detailed feedback about which part of the command is invalid and how to fix it.
+**person list** | A list of people and their details.
+**overview** | Provides an overview of the total number of cases and cases per category.
+
+<div style="page-break-after: always;"></div>
+Here is the breakdown of an individual *person card*.
```

Comment:
```suggestion
Here is the breakdown of an individual **person card**.
```
This is how it currently appears:
<img width="442" alt="Screenshot 2023-04-07 at 11 10 13 AM" src="https://user-images.githubusercontent.com/110474344/230533354-5d32e20c-2de9-46ea-aeff-96a288d34f6a.png">

Reply:
Additionally, might it be better to standardise the use of "case" instead of "person" across all our documentation?

Reply:
okay i'll take note and update this image